examples/helper-function-basic/README.md

```diff
@@ -0,0 +1,62 @@
+# Example
+
+This application is created as [described here](https://vitejs.dev/guide/):
+
+```sh
+npm create vite@latest example -- --template react-ts
+```
+
+## Configure Vite
+
+We changed the `vite.config.ts` to add the generation of the manifest file and made sure to overwrite the main entry point. Here's how the `vite.config.ts` looks after the changes:
+
+```ts
+import react from '@vitejs/plugin-react'
+import { defineConfig } from 'vite'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  build: {
+    // generates .vite/manifest.json in outDir
+    manifest: true,
+
+    rollupOptions: {
+      // overwrite default .html entry
+      input: "/src/main.tsx",
+    },
+  },
+})
+```
+
+## Server side
+
+We then added the [`main.go`](https://github.com/olivere/vite/tree/main/examples/basic/main.go).
```

Comment:
@ge3224 / @olivere This link needs changing to point to `helper-function-basic` not `basic`. I don't have write permissions sorry

Reply:
@danclaytondev Thanks for catching that. I think I got it fixed, using a relative path, here in commit [4d03fba](https://github.com/olivere/vite/pull/10/commits/4d03fba77f3323ba0cfd583794b209078e69f3e6)